src/vs/workbench/browser/parts/editor/editor.contribution.ts

```diff
@@ -363,6 +363,9 @@ registry.registerWorkbenchAction(new SyncActionDescriptor(MoveGroupLeftAction, M
 registry.registerWorkbenchAction(new SyncActionDescriptor(MoveGroupRightAction, MoveGroupRightAction.ID, MoveGroupRightAction.LABEL, { primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KEY_K, KeyCode.RightArrow) }), 'View: Move Editor Group Right', category);
 registry.registerWorkbenchAction(new SyncActionDescriptor(MoveEditorToPreviousGroupAction, MoveEditorToPreviousGroupAction.ID, MoveEditorToPreviousGroupAction.LABEL, { primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.LeftArrow, mac: { primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyCode.LeftArrow } }), 'View: Move Editor into Previous Group', category);
 registry.registerWorkbenchAction(new SyncActionDescriptor(MoveEditorToNextGroupAction, MoveEditorToNextGroupAction.ID, MoveEditorToNextGroupAction.LABEL, { primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.RightArrow, mac: { primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyCode.RightArrow } }), 'View: Move Editor into Next Group', category);
+registry.registerWorkbenchAction(new SyncActionDescriptor(MoveEditorToFirstGroupAction, MoveEditorToFirstGroupAction.ID, MoveEditorToFirstGroupAction.LABEL, { primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KEY_1, mac: { primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyCode.KEY_1 } }), 'View: Move Editor into First Group', category);
```

Comment:
@shobhitchittora keybinding `KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KEY_1` is already taken by "Toggle Editor Layout" and cannot be used again

Reply:
Correct me if I'm wrong - 

1. The primary key binding scheme is applied to Windows and Linux builds. Mac has it's own primary binding object specified.
2. `KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KEY_1` is MAC-primary for "Toggle Editor Layout", while it's WIN/LINUX-primary for "MoveEditorToFirstGroupAction".
3. Any other suggestions?

Reply:
@shobhitchittora you are right, sorry I did not see the mac specific keybinding

Reply:
Thanks for looking into it 👍🏻.

Reply:
Done.